server/src/server/app.ts

```diff
@@ -32,6 +33,7 @@ app.use("/items", itemsRouters);
 app.use('/photos', photosRouters);
 app.use('/tts', ttsRouters);
 app.use('/user', UserRouters);
+app.use('/upload', S3UploaderRouters);
```

Comment:
I might consider moving `/upload` to be in `/photos` existing routes.
So the route would be `/photos/upload`.

Of course the content of `s3-bucket.routes.ts` will be moved to `photos.routes.ts`